plugins/org.eclipse.emf.mwe2.language/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,7 @@ Bundle-ActivationPolicy: lazy
 Require-Bundle: org.eclipse.emf.ecore,
  org.eclipse.emf.common,
  org.eclipse.emf.mwe2.runtime;bundle-version="2.9.2",
- org.objectweb.asm;bundle-version="[5.0.1,7.0.0)";resolution:=optional,
+ org.objectweb.asm;bundle-version="[5.0.1,8.0.0)";resolution:=optional,
```

Comment:
Since we depend on Xtext 2.14, it would be reasonable to use the very same lower bound, wouldn't it? [6.2.1, ..]

Reply:
you mean 6.1.1? 

`'asm': '6.1.1',`